src/elf.rs

```diff
@@ -564,9 +564,13 @@ impl<E: UserDefinedError, I: InstructionMeter> Executable<E, I> {
         }
 
         for section_header in elf.section_headers.iter() {
-            if let Some(this_name) = elf.shdr_strtab.get_at(section_header.sh_name) {
-                if this_name.starts_with(".bss") {
-                    return Err(ElfError::BssNotSupported);
+            if let Some(name) = elf.shdr_strtab.get_at(section_header.sh_name) {
+                if name.starts_with(".bss")
+                    || (name.starts_with(".data")
```

Comment:
This new stricter part of the condition needs to be configurable here and feature gated in the mono repo. Take a look [here](https://github.com/solana-labs/rbpf/blob/77b89722eac6fed0da8d362739ba59649ba6e9c9/src/vm.rs#L196) and [here](https://github.com/solana-labs/solana/blob/f56eb7f882432ffbb0ef7c7279a33d765f9f30ad/programs/bpf_loader/src/lib.rs#L100) to get an idea of what I mean by that.

Reply:
Oh yup makes sense